recipes/pdspy/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "pdspy" %}
+{% set version = "1.5.0" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: 94d68282132f0272ef41fa4da84eebbf859ead2f3cb407223feeb9f2d1a7a247
+
+build:
+    number: 0
+    script: "{{ PYTHON }} -m pip install . -vv --no-deps"
+    skip: True  # [win]
+
+requirements:
+    build:
+        - {{ compiler('c') }}
+        - {{ compiler('fortran') }}
+    host:
+        - python
+        - numpy
+        - cython
+        - pip
+    run:
+        - python
+        - {{ pin_compatible('numpy') }}
+        - scipy
+        - matplotlib-base
+        - astropy
+        - h5py
+        - mpi4py
+        - emcee
+        - dynesty
+        - corner
+        - galario
+        - schwimmbad
+
+test: {}
```

Comment:
This needs a real test, see the example dir and the documentation

Reply:
Is the included run_test.sh file insufficient, or is it specified incorrectly? My understanding was that such a script is detected automatically. I couldn't find anything in the Conda-forge docs about this... I originally had import: pdspy here, but that led to issues with importing mpi4py in the test environment, so I'm now following the mpi4py example to set some environment variables to get the test happily running. Thank you for the help!

Reply:
Yeah, the run script helps, but you probably should have a simple import test or existence test too

Reply:
I've added a test that one of the included script exists, and the import test happens in run_test.sh. Is this ok? Thanks!